### PR TITLE
Change the echo command

### PR DESCRIPTION
**Pull request by Israel-Laguan**
# Overview  
This PR change the echo command for send-message that allow set title and description in a Embed message

## How to test  
1. Run the bot with yarn start
2. Select the slash command /send-message
3. Set a title and description to send and select a channel (optional)

### CODE REVIEW

File: src/commands/utility/send-message.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('send-message')
+    .setDescription('Replies with your input!')
+    .addChannelOption((option) =>
+      option.setName('channel').setDescription('The channel to message into')
+    ),
+  async execute(interaction) {
+    const channel = interaction.options.getChannel('channel');
+    const channelId = channel?.id;
```
Comment (Israel-Laguan) [inline src/commands/utility/send-message.js:13]: What would happen if channel id is not found? not sure if we can add a default channel?

File: src/modals/send-message-modal.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    .setLabel(' Message Title')
+    .setStyle(TextInputStyle.Short)
+    .setRequired(true);
+
+  const descriptionInput = new TextInputBuilder()
+    .setCustomId('descriptionInput')
+    .setLabel('Message Description')
+    .setStyle(TextInputStyle.Paragraph)
+    .setRequired(true)
+    .setMinLength(50);
```
Comment (Israel-Laguan) [inline src/modals/send-message-modal.js:24]: Not sure if this minimum can be show in the UI so the user knows why adding less characters fails?

Comment (YoelFerreyra) [reply src/modals/send-message-modal.js:24]: The modal of discord add the minimum in the ui by default

File: src/commands/utility/send-message.js (hunk 1 above)
Comment (YoelFerreyra) [reply src/commands/utility/send-message.js:13]: The channel came form the slash command. If the user pass the channel the logic enter in the first conditional in the other hand the message will send to the same channel that open it

File: src/commands/utility/send-message.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const { SlashCommandBuilder } = require('discord.js');
+const { createSendMessageModal } = require('../../modals/send-message-modal');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('send-message')
+    .setDescription('Replies with your input!')
```
Comment (heliomar-pena) [inline src/commands/utility/send-message.js:7]: The description can change now. Something like: `Send a message with the bot`?

File: src/utils/send-message-formatter.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+function formatSendMessage({ title, description, userName }) {
+  return `
+    # ${title}
+
+    ${description}
+
+    > Anuncio: ${userName}
+    `;
```
Comment (heliomar-pena) [inline src/utils/send-message-formatter.js:8]: estaba pensando en algo asi para que los campos puedan ser opcionales
```suggestion
    ${title ? `# ${title}` : ''}
    ${description ? `\n\n${description}` : ''}
    > Anuncio: ${userName}
    `;
```

Comment (YoelFerreyra) [reply src/utils/send-message-formatter.js:8]: image
The change is ready

**Review by YoelFerreyra — COMMENTED** [review]
Approve

**Review by YoelFerreyra — APPROVED** [review]
Approve

**Review by heliomar-pena — APPROVED** [review]

**Pull request merged**